### DISCUSSION TITLE
"style" instead of "build" in the hero section

### DISCUSSION THREAD

**Discussion by Vanilagy** [tailwindlabs/tailwindcss#10203] [Ideas]
Opening the TailwindCSS website greets the user with this message:
<img width="987" alt="CleanShot 2022-12-31 at 19 16 32@2x" src="https://user-images.githubusercontent.com/1696106/210152357-eb31989a-c499-482e-a95f-ae877f1eef68.png">

Which sounds nice, but if you think about it, is quite meaningless without context. "Building websites" is an incredibly generic expression and does not really further narrow down what Tailwind is about. It only gets somewhat narrowed in when you read "without ever leaving your HTML". But then again, that could also be Vue. Or Svelte. Or just regular HTML. Those also let you build websites without leaving HTML.

I'd say, for the sake of clarity, we should be saying "Rapidly *style* modern websites without ever leaving your HTML." Or rapidly build modern designs. That makes it instantly clear what Tailwind really is about, and clarity should be central when thinking about how new users discover your product.

**Comment by yanggggjie** [comment]
Hi, I think maybe not noly developer use taildwindcss and some designer use it too."An API for your design system." it says in docs.(sorry for my poor English)

**Comment by Vanilagy** [comment]
Yeah of course. But it wouldn't hurt to let designers know that a website is about their field of expertise.

Also, come to think of it, the word "modern" is equally as incorrect. The whole point of Tailwind is that it's so low-level that it's basically unopinionated, so you can use it to design any style of website you want - not just modern stuff. Thing is, it *sounds* nice. So how do you balancing marketing speak with being factually correct? Maybe something like "beautiful" would evoke more emotions while also being less specific about its objective meaning.
